### PR TITLE
Add Home Kit Paper Instruction Page

### LINKED ISSUE #732: !! Nationwide launch: additional public page that is unlinked
For the home kit paper instruction, we have a QR on the top right that will be mapping to a video instruction both in English and Spanish from Columbia on how to collect the blood sample.  

Screen Shot 2021-01-12 at 11 14 15 AM

Please create a public page (no login requirement) with the following URL: 

https://covidrecoverycorps.org/testkit

English Instructions: 

https://youtu.be/--MX5u9a12Y

Spanish instructions: 
https://youtu.be/bM6MoVTAegQ

### PR DESCRIPTION
**Pull request by leonli33**
Created a new page with path "/testkit" that displays a video to the user about home kit paper instructions.

The page looks as follows:
image

Resolves #732

### CODE REVIEW

File: src/components/static/TestKitInformationScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
```
Comment (conundrumgirl) [inline src/components/static/TestKitInformationScreen.tsx:1]: One general note is that this is responsive site so it should look OK no mobile. Please ping  me if you have implementation questions about it

File: src/components/static/TestKitInformationScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      <div className={classes.topBanner}>
+        <NavLink to="/home">
+          <CovidRecoveryCorpsLogo />
+        </NavLink>
+      </div>
+
+      <h2>Blood Sample Collection Instructions</h2>
+      <div className={classes.languageRow}>
+        <div
+          className={engishTextClasses.join(' ')}
```
Comment (conundrumgirl) [inline src/components/static/TestKitInformationScreen.tsx:73]: In general for conditional classes it's easier to use template strings `${}` rather then manipulating an array and joining it

File: src/components/static/TestKitInformationScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          English
+        </div>
+        <div
+          className={spanishTextClasses.join(' ')}
+          onClick={() => handleLanguagePressed('SPANISH')}
+        >
+          en español
+        </div>
+      </div>
+      <Iframe
```
Comment (conundrumgirl) [inline src/components/static/TestKitInformationScreen.tsx:85]: do you need react-iframe? can you just use regular iframe? Another approach to this is just rendering different iframes depending on englishIsSelected.

File: src/components/static/TestKitInformationScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+      <h2>Blood Sample Collection Instructions</h2>
+      <div className={classes.languageRow}>
+        <div
+          className={engishTextClasses.join(' ')}
+          onClick={() => handleLanguagePressed('ENGLISH')}
+        >
+          English
+        </div>
+        <div
```
Comment (conundrumgirl) [inline src/components/static/TestKitInformationScreen.tsx:78]: It is preferable for an element that is used as a button to be an actual button rather then the div styled to look as one

File: src/components/static/TestKitInformationScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+const TestKitInformationScreen: React.FC<{}> = props => {
+  const [englishIsSelected, setEnglishSelected] = useState<boolean>(true)
+  const classes = useStyles()
+  const engishTextClasses = [classes.languageText]
+  const spanishTextClasses = [classes.languageText]
+  englishIsSelected
+    ? engishTextClasses.push(classes.selectedLanguage)
+    : spanishTextClasses.push(classes.selectedLanguage)
+
```
Comment (conundrumgirl) [inline src/components/static/TestKitInformationScreen.tsx:53]: how about something like this const class = `${classes.languageText} ${classes.selectedLanguage}`

**Review by conundrumgirl — CHANGES_REQUESTED** [review]

File: src/components/static/TestKitInformationScreen.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+  return (
+    <div className={classes.container}>
+      <div className={classes.topBanner}>
+        <NavLink to="/home">
+          <CovidRecoveryCorpsLogo />
+        </NavLink>
+      </div>
+      <h2 className={classes.header}>{`${
+        language === 'en'
```
Comment (conundrumgirl) [inline src/components/static/TestKitInformationScreen.tsx:78]: the text should be in the translation files  (locales/../translation) and then you would use useTranslate hook. It checks the current language for you and selects the appropriate location. You can find usage example here: https://github.com/Sage-Bionetworks/covidrecoverycorps/blob/master/src/components/learningHub/LearningHub.tsx#L233

**Review by conundrumgirl — CHANGES_REQUESTED** [review]
Looks good. Just one change needed for how to handle the text.

File: src/components/static/TestKitInformationScreen.tsx (hunk 6 above)
Comment (leonli33) [reply src/components/static/TestKitInformationScreen.tsx]: Got it!

**Review by conundrumgirl — APPROVED** [review]
Looks good!

**Pull request merged**